tests/test-distro.sh

```diff
@@ -117,3 +117,23 @@ lxc exec $NAME -- microk8s enable community
 lxc exec $NAME -- script -e -c "pytest -s /var/snap/microk8s/common/addons/community/tests/test-addons.py"
 lxc exec $NAME -- microk8s reset
 lxc delete $NAME --force
+
+if [[ ${TO_CHANNEL} =~ /.*/microk8s.*snap ]]
+then
+  snap install ${TO_CHANNEL} --dangerous --classic
+else
+  snap install microk8s --channel=${TO_CHANNEL} --classic
+fi
+
+microk8s status --wait-ready
+
+if [ -d "/var/snap/microk8s/common/addons/eksd" ]
+then
+  pytest -s /var/snap/microk8s/common/addons/eksd/tests/test-addons.py
+fi
+
+if [ -f "/var/snap/microk8s/common/addons/core/tests/test-addons.py" ] &&
+   grep test_gpu /var/snap/microk8s/common/addons/core/tests/test-addons.py -q
+then
+  timeout 3600 pytest -s /var/snap/microk8s/common/addons/core/tests/* -k gpu
```

Comment:
I'd prefer
```suggestion
  timeout 3600 pytest -s /var/snap/microk8s/common/addons/core/tests/test-addons.py -k test_gpu
```
 if there isn't a specific reason to use wildcard.